stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -93,6 +95,26 @@ internal class PaymentOptionsAdapter(
         ).firstOrNull() ?: NO_POSITION
     }
 
+    /**
+     * Find the index of [paymentSelection] in the current items. Return -1 if not found.
+     */
+    private fun findSelectedPosition(paymentSelection: PaymentSelection?): Int {
+        return items.indexOfFirst { item ->
```

Comment:
If you check paymentSelection first for null you won't have to iterate through the entire loop.

Reply:
Done